### PR TITLE
Update quickstarts with latest syntax.

### PR DESCRIPTION
**Pull request by technomancy**
Had this committed to the old repo earlier.

### CODE REVIEW

**Comment by smart-alek** [comment]
LGTM

File: jekyll/_cci2/quickstart-javascript.md [hunk 1]
```diff
@@ -54,42 +54,43 @@ jobs:
...
     docker:
-      image: node:7.4.0
+      - image: node:7.4.0
     steps:
-      - shell:
-        name: Pre-Dependencies
-        command: mkdir ~/artifacts
+      - run:
+          name: Pre-Dependencies
+          command: mkdir /home/ubuntu/cci-demo-react/artifacts
```
Comment (smart-alek) [inline jekyll/_cci2/quickstart-javascript.md]: Should we change this back to `~/cci-demo-react/artifacts`? Or do we want to leave the `/home/ubuntu`?

File: jekyll/_cci2/quickstart-javascript.md [hunk 2]
```diff
@@ -54,42 +54,43 @@ jobs:
...
           command: npm test
 ```
 
 Remember when we created a directory for our artifacts? CircleCI won't unless we tell it where to look.
 
 ```yaml
-      - artifacts_store:
-          path: ~/artifacts
+      - store_artifacts:
+          path: /home/ubuntu/cci-demo-react/artifacts
```
Comment (smart-alek) [inline jekyll/_cci2/quickstart-javascript.md]: Same question about the root directory.

File: jekyll/_cci2/quickstart-javascript.md (hunk 1 above)
Comment (technomancy) [inline jekyll/_cci2/quickstart-javascript.md]: There are problems with `~` in local builds for the top-level `working_directory` setting in a job (not in steps) but these were coming from me trying to debug that failure and forgetting to change it back. I'll bring back the tilde here.

**Pull request merged**